nixos/modules/programs/fzf.nix

```
@@ -5,18 +5,8 @@ let
 in {
   options = {
     programs.fzf = {
-      fuzzyCompletion = mkOption {
-        type = types.bool;
-        description = lib.mdDoc "Whether to use fzf for fuzzy completion";
-        default = false;
-        example = true;
-      };
-      keybindings = mkOption {
-        type = types.bool;
-        description = lib.mdDoc "Whether to set up fzf keybindings";
-        default = false;
-        example = true;
-      };
+      fuzzyCompletion = mkEnableOption (mdDoc "use fzf for fuzzy completion");
+      keybindings = mkEnableOption (mdDoc "set up fzf keybindings");
```

```suggestion
            keybindings = mkEnableOption (mdDoc "fzf keybindings");
```
`mkEnableOption` automatically prepends "Whether to enable"